ui/src/BulkUploads.js

```diff
@@ -0,0 +1,322 @@
+          Uploaded Bulk Process Files
+        </Typography>
+        {this.state.authorisedActivities.includes(
+          "VIEW_BULK_REFUSAL_PROGRESS"
+        ) && (
+          <>
+            <Typography variant="h6" color="inherit" style={{ marginTop: 20 }}>
+              Bulk Refusals
+            </Typography>
+            <TableContainer component={Paper}>
+              <Table>
+                <TableHead>
+                  <TableRow>
+                    <TableCell>File Name</TableCell>
+                    <TableCell>Date Uploaded</TableCell>
+                    <TableCell align="right">Status</TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>{bulkRefusalTableRows}</TableBody>
+              </Table>
+            </TableContainer>
+          </>
+        )}
+        {this.state.authorisedActivities.includes("LOAD_BULK_REFUSAL") && (
+          <>
+            <input
+              accept=".csv"
+              style={{ display: "none" }}
+              id="contained-button-file"
+              type="file"
+              onChange={(e) => {
+                this.handleBulkRefusalUpload(e);
+              }}
+            />
+            <label htmlFor="contained-button-file">
+              <Button
+                variant="contained"
+                component="span"
+                style={{ marginTop: 10 }}
+              >
+                Upload Bulk Refusal File
+              </Button>
+            </label>
+          </>
+        )}
+        <Dialog open={this.state.uploadInProgress}>
+          <DialogContent style={{ padding: 30 }}>
+            <Typography variant="h6" color="inherit">
+              Uploading file...
+            </Typography>
+            <LinearProgress
+              variant="determinate"
+              value={this.state.fileProgress * 100}
+              style={{ marginTop: 20, marginBottom: 20, width: 400 }}
+            />
+            <Typography variant="h6" color="inherit">
+              {Math.round(this.state.fileProgress * 100)}%
+            </Typography>
+          </DialogContent>
+        </Dialog>
+        <Snackbar
+          open={this.state.fileUploadSuccess}
+          autoHideDuration={6000}
+          onClose={this.handleClose}
+          anchorOrigin={{
+            vertical: "bottom",
+            horizontal: "left",
+          }}
+        >
+          <SnackbarContent
+            style={{ backgroundColor: "#4caf50" }}
+            message={"File upload successful!"}
+          />
+        </Snackbar>
+        <JobDetails
+          jobTitle={"Bulk Refusal"}
+          job={selectedJob}
+          showDetails={this.state.showDetails}
+          handleClosedDetails={this.handleClosedDetails}
+          onClickAway={this.handleClosedDetails}
+          onProcessJob={this.onProcessJob}
+          onCancelJob={this.onCancelJob}
+          authorisedActivities={this.state.authorisedActivities}
+          loadJobType={"BULK_REFUSAL"}
```

Comment:
I think you can safely leave job type out of it and just check the permission... we can clearly see that only `BULK_REFUSAL` job types would ever be supplied to this component